allennlp/data/dataset_readers/reading_comprehension/triviaqa.py

```diff
@@ -1,32 +1,245 @@
+# pylint: disable=no-member
```

Comment:
Where is this an issue?  Does it make sense to put it more locally?  I'm guessing this is from `sklearn` - you could also whitelist this in `.pylintrc`, and it might fix the issue.